src/ListView/ListView.stories.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { storiesOf, action } from '@kadira/storybook'
+import { withKnobs, boolean } from '@kadira/storybook-addon-knobs'
+import { defaultTemplate } from '../../storybook/decorators/storyTemplates'
+import MockListManager from './__mocks__/MockListManager'
+import ListView from './ListView'
+import ListViewItem from './ListViewItem'
+import ListViewItemContainer from './ListViewItemContainer'
+import ListViewItemExpansion from './ListViewItemExpansion'
+
+const stories = storiesOf('ListView', module)
+stories.addDecorator(withKnobs)
+stories.addDecorator(
+  defaultTemplate({
+    title: 'List View',
+    documentationLink: 'http://www.patternfly.org/pattern-library/content-views/list-view/'
+  })
+)
+
+stories.addWithInfo(
+  'basic list with expansion',
+  `This is the List View with basic expansion.`,
+  () => <MockListManager />
+)
+
+stories.addWithInfo(
+  'basic list with expansion knobs',
+  `This is the List View with basic expansion.`,
+  () => {
+    const firstLevelExpanded = boolean('First Level Expanded', false)
+    const secondLevelExpanded = boolean('Second Level Expanded', false)
+    return (
+      <ListView>
+        <ListViewItem isActive={firstLevelExpanded}>
+          <ListViewItemContainer
+            isExpansionItem
+            itemClicked={action('first level itemClicked')}
+            isActive={firstLevelExpanded}
+          >
+            First Level
+          </ListViewItemContainer>
+          <ListViewItemExpansion isActive={firstLevelExpanded}>
```

Comment:
@waldenraines thanks for the feedback and thanks for joining in! My personal thought is that the accordion is a different pattern (and we could surely include this as a new component in pf-react). The list view is catered for other things in our current PF design framework, and the new list view also supports nested lists (as the PR shows). Modifying accordion to do this would likely not be easy...and we already have a PF design for nested list views now.

With that being said, I hear @ohadlevy and @matanwerbner 's feedback. In this state, it would require for more effort downstream to track and extend the expansion state. I'm fine w/  developing this as a stateful component (just wanted to touch on this based on previous conversations w/ @jtomasek about stateless). In general, i think we should still prefer stateless when possible, but this is a case where maintaining state internally (and exposing it as needed) is probably a better choice.

@jtomasek thoughts? If we agree, I'm happy to expand on your API in Triple O and make this stateful (where necessary) and work w/ our new nested list view pattern.

thanks all for the feedback on this...